b2sdk/bucket.py

```diff
@@ -687,10 +687,10 @@ def copy(
         """
 
         copy_source = CopySource(file_id, offset=offset, length=length)
-        if length is None:
+        if not length:
```

Comment:
This check is not the same when `length` is 0.  I think this first branch should still happen in that case.

Reply:
The first branch is using a simple copy manager and the second emerger created by @mzukowski-reef 
The problem is that emerger doesn't work when copying 0-length files and actually is not needed for such files